server/src/test/java/org/elasticsearch/index/shard/IndexShardTests.java

```diff
@@ -4497,7 +4497,7 @@ public void testSupplyTombstoneDoc() throws Exception {
         closeShards(shard);
     }
 
-    public void testResetEngine() throws Exception {
+    public void resetEngineToGlobalCheckpoint() throws Exception {
```

Comment:
I think that the test name should be prefixed by `test` otherwise it won't be executed?